pydantic/_internal/_typing_extra.py

```diff
@@ -403,28 +427,13 @@ def get_type_hints(  # noqa: C901
                     is_argument=not isinstance(obj, types.ModuleType),
                     is_class=False,
                 )
-            value = typing._eval_type(value, globalns, localns)  # type: ignore
+            value = eval_type_backport(value, globalns, localns)
             if name in defaults and defaults[name] is None:
                 value = typing.Optional[value]
             hints[name] = value
         return hints if include_extras else {k: typing._strip_annotations(t) for k, t in hints.items()}  # type: ignore
 
 
-if sys.version_info < (3, 9):
```

Comment:
why don't we need this anymore?

Reply:
AFAICT this could always have been replaced with `typing._eval_type`. Now it has to be, with the backport.